.github/workflows/dry-run.yml

```diff
@@ -72,6 +72,17 @@ jobs:
           package: theming-core
           artifact_path: theming-*.tgz
           download_dependencies: true
+  buildComponentToolkit:
+    name: Build component-toolkit
+    runs-on: ubuntu-latest
+    needs:
+      - buildBrowserTestTools
+    steps:
+      - uses: cloudscape-design/.github/.github/actions/build-package@main
+        with:
+          package: component-toolkit
+          artifact_path: theming-*.tgz
```

Comment:
Copy paste mistake?

Reply:
Looks like it - thanks!

Reply:
And it still passed the build with incorrect line. Interesting 🤔